src/main/java/driver/SingletonDriver.java

```diff
@@ -1,5 +1,31 @@
 package driver;
 
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.util.concurrent.TimeUnit;
+
 public class SingletonDriver {
+    private static final String DRIVER_PATH = "src/main/resources/chromedriver.exe";
+    private static WebDriver driver;
+
+    private SingletonDriver() {
+        System.setProperty("webdriver.chrome.driver", DRIVER_PATH);
+        driver = new ChromeDriver();
+    }
+
+    public static WebDriver getWebDriverInstance() {
+        if (driver == null) {
+            new SingletonDriver();
+        }
+        return driver;
+    }
+
+    public static WebDriver openPage(WebDriver driver, String url) {
+        driver.get(url);
+        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
```

Comment:
Please move this and the next line to the SingletonDriver's constructor - it is more related to configuring WebDriver

Reply:
done